### PR TITLE
feat- add 2023 sprints page

### PR DESCRIPTION
**Pull request by SivanYeh**
## Description
Update page related to sprints and upload to 2023 website

## Steps to Test This Pull Request
1. click events/sprints in header
2. check if En/Zh content updated to 2023 information at Google [Doc](https://docs.google.com/document/d/1jZ9v_Fsc0dgWFB7TmQ8b4mclJBkqPoc9bwjQ9sdU12I/edit#heading=h.z437yptlh9hp) 

## Expected behavior
image
image

## Related Issue

#427

### CODE REVIEW

**Comment by SivanYeh** [comment]
wait for hackmd and kktix links

File: i18n/events/sprints.i18n.js [hunk 1]
```diff
@@ -4,55 +4,53 @@ export default genI18nMessages({
...
             'You can find the project list on {hackmd}.',
         ],
-        form: 'this Google Form',
+        form: 'Google Form',
     },
     'zh-hant': {
         title: '衝刺開發 (Sprints)',
         intro:
-            'PyCon APAC 2022 夏季衝刺開發的活動時間是 8/14 (日) 09:30 ~ 17:00。' +
+            'PyCon TW 2023 夏季衝刺開發的活動時間是 8/20 (日) 10:00 ~ 17:00。' +
```
Comment (yungshenglu) [inline i18n/events/sprints.i18n.js:38]: 「夏季」可以拿掉

File: i18n/events/sprints.i18n.js [hunk 2]
```diff
@@ -4,55 +4,53 @@ export default genI18nMessages({
     'en-us': {
         title: 'Sprints',
         intro:
-            'The Summer Sprint event will be held on 8/14 (Sun.) 09:30 ~ 17:00 (TST).\n\n' +
-            'A sprint event gathers open source project owners, contributors, ' +
-            'and people who want to contribute but are trying to find a place to start. ' +
-            'During a sprint, project hosts bring their unresolved issues or ' +
-            'new features under development and share with everyone. ' +
+            'The Summer Sprint event will be held on 8/20 (Sun.) 10:00 ~ 17:00 (TST).\n\n' +
```
Comment (yungshenglu) [inline i18n/events/sprints.i18n.js:7]: "Summer" can be removed.

**Review by yungshenglu — COMMENTED** [review]
建議可以把地點加上去

**Comment by SivanYeh** [comment]
Done!

**Review by yungshenglu — APPROVED** [review]
LGTM

**Pull request merged**